include.xml

```diff
@@ -2,11 +2,12 @@
 <project>
 	<icon path="assets/images/logo/HaxeFlixel.svg" />
 	
-	<assets path="assets/sounds" include="*.ogg" if="html5" unless="unit-test" />
+	<assets path="assets/sounds" include="*.m4a" if="html5" unless="unit-test" />
 	<assets path="assets/sounds" include="*.mp3" if="flash" embed="true" />
-	<assets path="assets/sounds" include="*.ogg" unless="flash || html5" embed="true" />
+	<assets path="assets/sounds" include="*.ogg" if="native" embed="true" />
+	
 	<assets path="assets/fonts/nokiafc22.ttf" embed="true" />
-	<assets path="assets/fonts/arial.ttf" embed="true" unless="FLX_NO_DEBUG" />
+	<assets path="assets/fonts/montserrat-Regular.ttf" embed="true" if="debug" />
```

Comment:
Is the `if=debug` here safe? `!FLX_NO_DEBUG` isn't the same as `debug`, meaning you could have a flixel debugger in release mode if you wanted - in which case the font would be missing.